添加条目：没有安装重复 Mod 却提示安装了重复 Mod

wuliaodexiaoluo opened the pull request:
~失踪人口回归~

wuliaodexiaoluo committed:
Update README.md

Deep-Dark-Forest commented on README.md:
```diff
@@ -134,6 +134,7 @@
  | 启动 Minecrat 1.7.10-1.12.2 游戏内无声音 | 此为 OpenAL 初始化失败所致，请在游戏启动后按下 F3+T 来重新初始化 OpenAL。 | #3779 [MC-9974](https://bugs.mojang.com/browse/MC-9974)
  | 因感染蠕虫病毒导致 PCL 崩溃或无法更新等 | *Collaborator 请注意，日志可见：文件名或文件夹名前含有 `cache`、`HD` 字样。* | #2964 #2970 #3204
  | 日志出现 not xxx in java.library.path | Java 不完整，非 PCL 问题，如果可能，请您自行准备 Java 而不是使用自动补全 Java 功能。 | #3496
+ | 没有安装重复模组却提示安装了重复模组 | 部分优化模组使用了相同的模组 ID 导致模组加载器误判为相同 Mod，请在安装优化模组前 **仔细核对官方文档/百科** 确认可以同时安装后再安装。| 4376
```
```suggestion
 | 没有安装重复模组却提示安装了重复模组 | 部分优化模组使用了相同的模组 ID 导致模组加载器误判为相同 Mod，请在安装优化模组前 **仔细核对官方文档/百科** 确认可以同时安装后再安装。| #4376
```

wuliaodexiaoluo committed:
Update README.md

CO-Author-By: Deep-Dark-Forest <131975043+Deep-Dark-Forest@users.noreply.github.com>

Co-authored-by: 深黑森林 <131975043+Deep-Dark-Forest@users.noreply.github.com>

WorldHim commented:
其实 mod id 重复也有可能是某个模组内置了另一模组
e.g. create 高版本内置 flywheel

wuliaodexiaoluo commented:
>其实 mod id 重复也有可能是某个模组内置了另一模组
e.g. create 高版本内置 flywheel

~Mod 作者一定在玩原神~

WForst-Breeze commented on README.md:
```diff
@@ -134,6 +134,7 @@
  | 启动 Minecrat 1.7.10-1.12.2 游戏内无声音 | 此为 OpenAL 初始化失败所致，请在游戏启动后按下 F3+T 来重新初始化 OpenAL。 | #3779 [MC-9974](https://bugs.mojang.com/browse/MC-9974)
  | 因感染蠕虫病毒导致 PCL 崩溃或无法更新等 | *Collaborator 请注意，日志可见：文件名或文件夹名前含有 `cache`、`HD` 字样。* | #2964 #2970 #3204
  | 日志出现 not xxx in java.library.path | Java 不完整，非 PCL 问题，如果可能，请您自行准备 Java 而不是使用自动补全 Java 功能。 | #3496
+ | 没有安装重复模组却提示安装了重复模组 | 部分优化模组使用了相同的模组 ID 导致模组加载器误判为相同 Mod，请在安装优化模组前 **仔细核对官方文档/百科** 确认可以同时安装后再安装。| #4376
```
可以搁在「无法加载 Mod 图标」下方

另，为了统一，模组 → Mod

wuliaodexiaoluo committed:
Update README.md

WForst-Breeze committed:
Optimize Expressions
